ios/browser/api/url_sanitizer/url_sanitizer_service.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_URL_SANITIZER_URL_SANITIZER_SERVICE_H_
+#define BRAVE_IOS_BROWSER_API_URL_SANITIZER_URL_SANITIZER_SERVICE_H_
+
+#import <Foundation/Foundation.h>
+
+OBJC_EXPORT
+@interface URLSanitizerService : NSObject
+- (instancetype)init NS_UNAVAILABLE;
+
+/**
+ Sanitizes the given URL.
+
+ @param url The URL to be sanitized.
+ @return A sanitized NSURL object.
+ */
+- (NSURL*)sanitizedURL:(NSURL*)url;
```

Comment:
`net::NSURLWithGURL` [can return nil](https://source.chromium.org/chromium/chromium/src/+/main:net/base/mac/url_conversions.mm;l=17;bpv=1;bpt=1), so you should make this `nullable`

Reply:
Done